_posts/2020-06-14-Emendations to Source Editions on IrishGen.md

```diff
@@ -0,0 +1,51 @@
+
+Occasionally, however, we encounter incongruities in existing editions and clarify the situation by checking online images of the manuscript. These images are available via repositories like [ISOS](https://www.isos.dias.ie) or [Digital.Bodleian](https://digital.bodleian.ox.ac.uk/). If the edition turns out to be in serious error, we follow the original manuscript in our encoding. This blogpost describes one such eventuality.
+
+## The Anachronistic Ancestor of Dál nAraide
+Cenél Lóegaire were a minor southern Uí Néill dynasty based around modern-day [Trim](https://www.logainm.ie/1416699.aspx ), Co. Meath. The genealogists traced their ancestry back to [Lóegaire](https://doi.org/10.1093/ref:odnb/15872), son of Níall Noígiallach, the pagan king of Tara whose druids confronted St. Patrick.
+
+In the CELT edition of the Book of Leinster, [the pedigree](https://celt.ucc.ie//published/G800011F/index.html) of their royal line contains an account of Fiacha Araide. He is the ancestor figure of the Ulster polity of Dál nAraide, based in Mag nEilne, the area just east of [Coleraine](https://www.logainm.ie/1416661.aspx).
+
+> Cu Ulad m Domnaill m Gillai Ultain m Oengusa m Cainelbain m Mael Chróin m Domnaill m Con Rai m Oengusa m Fheradaig Araide Bibre in cáinte de Mumnechaib ba sé ba rectaire do Chormac .h. Chuind. Cairech a ben. Is i ro anacht Fiacha mc Oengusa. inde dicitur Fiacha Araide a quo Dal Araide. m Mael Duin Dergainig m Colmáin m Aeda m Libuir m Dalleni m Ennai m Loegaire m Neill Noigiallaig.
+> > Cú Ulad, son of Domnall, son of Gilla Ultain, son of Óengus, son of Caindelbán, son of Mael Chróin, son of Domnall, son of Cú Rai, son of Óengus, son of Feradach (Araide Bibre. The satirist from among the Munstermen. He was steward to Cormac, grandson of Conn. Cairech was his wife. It was she who cared for Fiacha, son of Óengus. Thence he is called Fiacha Araide, whence are called the Dál nAraide), son of Máel Duin Dergainig, son of Colmán, son of Áed, son of Libur, son of Dallene, son of Enna, son of Lóegaire, son of Níall Noígiallach.
+
+This is impossible. Cormac, grandson of Conn, is Cormac mac Airt. He is Níall Noígiallach's legendary ancestor, placed five generations previous to him in the genealogies. Cormac's steward cannot be Níall's descendant, especially not by nine generations.
+
+## The Edition
+The passage quoted above is [the entry on Cenél Lóegaire](https://celt.ucc.ie//published/G800011F/index.html) from the edition of the Book of Leinster as digitised on CELT (the translation is my own). In [the printed edition](https://www.vanhamel.nl/codecs/Best,_et_al._1954-1983) (p. 1467), however, it is made clear that passage contains a section that has been added in the margin of the manuscript page. I have emphasised the inserted material in bold; it is presented in a footnote in the printed edition but placed after "m Fheradaig".
```

Comment:
I wonder if we should use Worldcat (https://www.worldcat.org/) permlinks so that people will be directed to the closest library to their location?  For instance: http://www.worldcat.org/oclc/264044957.

"I have emphasised the inserted material in bold; it is presented in a footnote in the printed edition but placed after "m Fheradaig"."

Maybe "I have emphasised the inserted material in bold; it is presented in a footnote in the printed edition but placed after "m Fheradaig" in the CELT file."

Reply:
On the other hand, CODECS provides more information on how the volume and page numbers work and on secondary sources and so forth. Is there any way we can somehow reference both?

The sentence you quote is about the printed edition. Would it be clearer if I changed it to something like this?

> "I have emphasised the inserted material in bold; in the printed edition, it is presented in a > footnote but the reference number for the footnote is still placed after "m Fheradaig"."

Reply:
True. I think it is OK to leave the CODECS page. I just wanted to bring up the possibility.

Yeah, that reads better.